addon/services/cookies.js

```diff
@@ -122,7 +121,20 @@ export default Service.extend({
 
     this._cacheFastBootCookie(...arguments);
 
-    responseHeaders.append('set-cookie', serializedCookie);
+    let replaced = false;
+    let existing = responseHeaders.getAll('set-cookie');
+
+    for (let i = 0 ; i < existing.length ; i++ ) {
+      if ( existing[i].startsWith(`${ name }=`) ) {
+        existing[i] = serializedCookie;
+        replaced = true;
+        break;
+      }
+    }
+
+    if ( !replaced ) {
+      responseHeaders.append('set-cookie', serializedCookie);
+    }
```

Comment:
I'm not sure I understand what this change does. Is this necessary?

Reply:
I didn't really intend to put this in the same PR because I haven't added tests for it yet.  So I can pull it out into a separate PR if you like.

But basically every time something writes a cookie right now it gets added to the header array, even if you've previously written a cookie of the same name.  ember-simple-auth in particular does this several times during a fastboot request, which results in sending a bunch of redundant info to the client.

So if you do 

```
cookies.write('key', 'value1');
cookies.write('key', 'value2');
```

The response headers are sent as

```
Set-Cookie: key=value1
Set-Cookie: key=value2
```

This change updates/replaces the existing header entry if there's already one for the name instead of always appending a new entry.